### PR TITLE
Fix sqlite shutdown race

### PR DESCRIPTION
**Pull request by moznuy**
Remove sqlite3_shutdown() 

Because:
1. According to sqlite3 docs sqlite3_initialize() and sqlite3_shutdown() do not have to be invoked
2. sqlite3_initialize() is never called explicilty
3. sqlite3_shutdown() is not threadsafe and sqlite_init_multithreaded is not called holding a lock
4. According to docs all connections must be closed before invoking sqlite3_shutdown() but they are not (from the different threads).

Basically my team were experiencing sudden loss of service after a while.
I started to look into it. On local machine I was getting:
1. SEGFAULT occasionally. (segfaults in sqlite lib)
2. loss of service
3. normal work

These changes removed for me first two points.

Maybe related to #730

### CODE REVIEW

**Comment by misi** [comment]
Many thanks! Sorry for my late reply!

**Comment by misi** [comment]
Many thanks for your PR and changes it is merged...
